tools/nixos/iso_installer.sh

```diff
@@ -0,0 +1,118 @@
+#!@shell@
+
+# SPDX-License-Identifier: MPL-2.0
+
+set -e
+
+echo "=== Asterinas NixOS Installer ==="
+echo "Default paths:"
+echo "  distro: @distroPath@"
+echo "  kernel: @kernelPath@"
+echo "  tools: @toolsPath@"
```

Comment:
How does printing these paths help the user of this ISO installer? They always point to folders under Nix stores. The hash values do not mean anything to the user.

In addition, these paths should NOT be called "default paths" because the script provide no means to override them.